build.ps1

```diff
@@ -58,31 +62,39 @@ if ($missingTools.Count -gt 0) {
 Push-Location $PSScriptRoot
 
 # Clean step
-if($Clean) {
+if($Clean.IsPresent) {
     git clean -fdx
 }
 
 # Build step
+if(!$NoBuild.IsPresent) {
+    # Install using PSDepend if it's available, otherwise use the backup script
+    if ((Get-Module -ListAvailable -Name PSDepend).Count -gt 0) {
+        Invoke-PSDepend -Path src -Force
+    } else {
+        & "$PSScriptRoot/tools/InstallDependencies.ps1"
+    }
 
-# Install using PSDepend if it's available, otherwise use the backup script
-if ((Get-Module -ListAvailable -Name PSDepend).Count -gt 0) {
-    Invoke-PSDepend -Path src -Force
-} else {
-    & "$PSScriptRoot/tools/InstallDependencies.ps1"
-}
-
-dotnet build -c $Configuration
-dotnet publish -c $Configuration
+    dotnet build -c $Configuration
+    dotnet publish -c $Configuration
 
-Push-Location package
-dotnet pack -c $Configuration
-Pop-Location
+    Push-Location package
+    dotnet pack -c $Configuration
```

Comment:
Same here, get rid of `push/pop-location` by `dotnet pack $PSScriptRoot/package`.

Reply:
fixed